AwsCryptographicMaterialProviders/runtimes/java/src/testExamples/java/software/amazon/cryptography/example/hierarchy/ExampleTests.java

```diff
@@ -24,35 +24,38 @@
 
 public class ExampleTests {
 
+  static final String hv2CreateTestPrefix = "create-hierarchy-version-2-java-";
```

Comment:
```suggestion
  static final String hv2CreateTestPrefix = "create-hierarchy-key-java-";
```

Small change, these tests don't only create HV2 keys

Reply:
Sorry, misread